lite-pm/styles/AppPage.module.css

```diff
@@ -0,0 +1,14 @@
+.h2 {
+    font-size: 19pt;
+    margin-top: 75pt;
+}
+
+.inputs{
+    border: none;
+    border-bottom: 0.1px solid black;
```

Comment:
`0.1px` will likely glitch out in different devices and screen sizes/zooms. Try `1px` or `0.5px`